packages/cli/src/modules/indexing/indexing.commands.ts

```diff
@@ -0,0 +1,17 @@
+import { Command } from 'commander';
+import { deployIndexer } from './indexing.service';
+
+export function createIndexingCommands(cli: Command) {
+  const indexing = cli
+    .command('indexing')
+    .description('Commands for deployment of indexers on Apillon platform');
+
+  indexing
+    .command('deploy-indexer')
```

Comment:
I would change this just to `deploy`, otherwise you have `apillon indexing deploy-indexer`, so the "-indexer" prefix is redundant